db/data/20211123143349_duplicate_trainee_records.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class DuplicateTraineeRecords < ActiveRecord::Migration[6.1]
+  def up
+    Trainee.where(trainee_id: [10653541, 10653500, 10653419]).each(&:destroy)
```

Comment:
We shouldn't rely on `trainee_id` as it's not guaranteed to be unique. Can we use the actual ID or the `dttp_id`?